<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgFilter</class>
 <widget class="QDialog" name="DlgFilter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>480</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Monospace</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Apply Filter</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/res/res/iconmonstr-filter-4-240.png</normaloff>:/res/res/iconmonstr-filter-4-240.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#DlgFilter{
background-color: #cccccc;
}

#grpbx_AddFilter{
border: 1px solid #3A3A3A;
border-radius:5px;
background-color:#f0f0f0;
}

QLabel{
color: blue;
}

QLineEdit{
background-color: #DDF1FF;
color: #0076C8;
padding-left: 10px;
}

QPushButton{
color: #ffffff;
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #5ddced, stop: 0.1 #59bfec, stop: 0.5 #5494eb, stop: 0.9 #517eea, stop: 1 #5077ea);
border-radius: 5;
font-weight:bold;
}
QPushButton:pressed{
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #5077ea, stop: 0.1 #517eea, stop: 0.5 #5494eb, stop: 0.9 #59bfec, stop: 1 #5ddced);
}
QPushButton:disabled{
background: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 1 #a7a7a7, stop: 0 #f4f4f4);
color:#9d9d9d;
}


QComboBox { 
border-radius:5px; 
background: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #5ddced, stop: 0.1 #59bfec, stop: 0.5 #5494eb, stop: 0.9 #517eea, stop: 1 #5077ea); 
padding: 1px 23px 1px 3px; 
min-width: 6em; 
color: black;
}
QComboBox::drop-down {
subcontrol-origin: padding; 
subcontrol-position: top right; 
width: 12px; 
border-top-right-radius: 5px; 
border-bottom-right-radius: 5px;
}
QComboBox QAbstractView{ 
margin-top: 3px;
background-color: #4f4f4f; 
color: #999999; 
selection-background-color: #999999; 
selection-color: #4f4f4f;
}
QComboBox::down-arrow { 
image: url(:/ico/res/cmbico.png);
}
QComboBox::disabled{
background: QLinearGradient(x1:0, y1:0, x2:0, y2:1, stop: 1 #A7A7A7, stop: 0 #F4F4F4);
color: #9D9D9D;
}
QComboBox QAbstractItemView::item{
margin-top: 5px;
}


QListView{
background-color: #DDF1FF;
alternate-background-color: #A1A1A1;
font-family: &quot;calibre&quot;;
font-style: normal;
font-size: 13px;
color: black;
}
QListView::item{
padding-top: 4px;
padding-bottom: 4px;
}
QListView::indicator{
width: 40px;
height: 18px;
}

/* !! Custom Scroll Bar !! */
QScrollBar:vertical {
background: #ededed; width: 8px; 
}
QScrollBar::handle:vertical {
background: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #5077ea,  stop: 1 #5ddced); min-height: 40px;
}
QScrollBar::handle:vertical:hover {
background: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 1 #0037db,  stop: 0 #0037db);
}
QScrollBar::add-line:vertical { 
background: #75a1ff; height: 0px; subcontrol-position: bottom;   subcontrol-origin: margin; 
}
QScrollBar::sub-line:vertical { 
background: #75a1ff; height: 0px; subcontrol-position: top;  subcontrol-origin: margin;
}

QScrollBar:horizontal {
background: #ededed; 
height:8px;
}
QScrollBar::handle:horizontal {
background: QLinearGradient( x1: 0, y1: 0, x2: 1, y2: 0, stop: 0 #5077ea,  stop: 1 #5ddced);
min-width: 40px;
}
QScrollBar::handle:horizontal:hover {
background: QLinearGradient( x1: 0, y1: 0, x2: 1, y2: 0, stop: 1 #0037db,  stop: 0 #0037db);
}
QScrollBar::add-line:horizontal {
background: #75a1ff;
width: 0px;
subcontrol-position: bottom; 
subcontrol-origin: margin;
}
QScrollBar::sub-line:horizontal {
background: #75a1ff;
width: 0px;
subcontrol-position: top;
subcontrol-origin: margin;
}
</string>
  </property>
  <widget class="QGroupBox" name="grpbx_AddFilter">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>8</y>
     <width>300</width>
     <height>194</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
border: 1px solid black\n
}</string>
   </property>
   <property name="title">
    <string>Select Filters</string>
   </property>
   <widget class="QComboBox" name="cmbbx_OpType">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>282</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbbx_OpSubj">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>74</y>
      <width>282</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_OpFormatStr">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>110</y>
      <width>278</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>------- format</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_OpFormat">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>150</y>
      <width>279</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QListView" name="lstview_Filters">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>208</y>
     <width>300</width>
     <height>210</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pshbtn_Add">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>137</y>
     <width>79</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pshbtn_Remove">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>173</y>
     <width>79</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>REMOVE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pshbtn_Load">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>209</y>
     <width>79</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>LOAD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pshbtn_Save">
   <property name="geometry">
    <rect>
     <x>316</x>
     <y>245</y>
     <width>79</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>SAVE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pshbtn_Apply">
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>434</y>
     <width>91</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pshbtn_Cancel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>434</y>
     <width>91</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
